tasks/cnm-to-cma/tests/test_cnm2cma.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import patch
```

Comment:
Consider using the pytest-mock mocker fixture https://pytest-mock.readthedocs.io/en/latest/usage.html